tools/devops/automation/templates/build/build-nugets.yml

```diff
@@ -0,0 +1,39 @@
+# all steps that are required to build the nugets and publish them
+
+steps:
+
+- bash:  $(Build.SourcesDirectory)/xamarin-macios/tools/devops/automation/scripts/bash/build-nugets.sh
```

Comment:
What happens if this produces at least one nuget, but still fails?

Reply:
It will continue to the next steps due to the continueOnError: true

Reply:
And then it looks like it will set the "Nugets built" status to "Success", or am I reading this wrong?

Reply:
We can remove the continueOnError and do an on failed() but the we need to add branching for the other code. I think that is better to do it in a diff PR since the code for building is just a move from the main build.yml to build-nugget.yml

But you are definitely right this is at least fishy and should have been fixed when we landed the 'Generate and Upload Build Asset Registry Manifest' step, because it is also going to return an error on a continueOnError :/

Reply:
Issue https://github.com/xamarin/maccore/issues/2419 I'll fix it in the morning.